src/app/layout.tsx

```diff
@@ -11,20 +15,31 @@ const geistMono = Audiowide({
   subsets: ["latin"],
   weight: "400",
 });
-export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
-};
+// export const metadata: Metadata = {
+//   title: "Create Next App",
+//   description: "Generated by create next app",
+// };
 
 export default function RootLayout({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const [language, setLanguage] = useState<"en" | "ru">("en");
+
+  const handleLanguageToggle = () => {
+    setLanguage((prevLanguage) => (prevLanguage === "en" ? "ru" : "en"));
```

Comment:
it is not scalable solution (imagine you will need to add third language). Use switch case for such purpose